Content/Library/docsSrc/content/style.css

```diff
@@ -0,0 +1,321 @@
+}
+
+.table thead td.fit, 
+.table th.fit {
+    white-space: nowrap;
+    width: 1%;
+}
+/*--------------------------------------------------------------------------
+  Formatting for page & standard document content
+/*--------------------------------------------------------------------------*/
+
+body {
+  font-family: 'Open Sans', serif;
+}
+
+pre {
+    word-wrap: inherit;
+}
+
+/* Format the heading - nicer spacing etc. */
+.masthead {
+  overflow: hidden;
+}
+.masthead .muted a {
+  text-decoration:none;
+  color:#999999;
+}
+.masthead ul, .masthead li {
+  margin-bottom:0px;
+}
+.masthead .nav li {
+  margin-top: 15px;
+  font-size:110%;
+}
+.masthead h3 {
+  margin-bottom:5px;
+  font-size:170%;
+}
+hr {
+  margin:0px 0px 20px 0px;
+}
+
+/* Make table headings and td.title bold */
+td.title, thead {
+  font-weight:bold;
+}
+
+/* Format the right-side menu */
+#menu {
+  margin-top:50px;
+  font-size:11pt;
+  padding-left:20px;
+}
+
+#menu .nav-header {
+  font-size:12pt;
+  color:#606060;
+  margin-top:20px;
+}
+
+#menu li {
+  line-height:25px;
+}
+
+.wrapper {
+    margin-top: -56px;
+    padding-top: 56px;
+}
+
+/* Change font sizes for headings etc. */
+#main h1 { font-size: 26pt; margin:10px 0px 15px 0px; font-weight:400; }
+#main h2 { font-size: 20pt; margin:20px 0px 0px 0px; font-weight:400; }
+#main h3 { font-size: 14pt; margin:15px 0px 0px 0px; font-weight:600; }
+#main p  { font-size: 11pt; margin:5px 0px 15px 0px; }
+#main ul { font-size: 11pt; margin-top:10px; }
+#main li { font-size: 11pt; margin: 5px 0px 5px 0px; }
+#main strong { font-weight:700; }
+
+/*--------------------------------------------------------------------------
+  Formatting for API reference
+/*--------------------------------------------------------------------------*/
+
+.type-list .type-name, .module-list .module-name {
+  width:25%;
+  font-weight:bold;
+}
+.member-list .member-name {
+  width:35%;
+}
+#main .xmldoc h2 {
+  font-size:14pt;
+  margin:10px 0px 0px 0px;
+}
+#main .xmldoc h3 {
+  font-size:12pt;
+  margin:10px 0px 0px 0px;
+}
+.github-link {
+  float:right;
+  text-decoration:none;
+}
+.github-link img {
+  border-style:none;
+  margin-left:10px;
+}
+.github-link .hover { display:none; }
+.github-link:hover .hover { display:block; }
+.github-link .normal { display: block; }
+.github-link:hover .normal { display: none; }
+
+/*--------------------------------------------------------------------------
+  Links
+/*--------------------------------------------------------------------------*/
+
+h1 a, h1 a:hover, h1 a:focus,
+h2 a, h2 a:hover, h2 a:focus,
+h3 a, h3 a:hover, h3 a:focus,
+h4 a, h4 a:hover, h4 a:focus,
+h5 a, h5 a:hover, h5 a:focus,
+h6 a, h6 a:hover, h6 a:focus { color : inherit; text-decoration : inherit; outline:none }
+
+/*--------------------------------------------------------------------------
+  Additional formatting for the homepage
+/*--------------------------------------------------------------------------*/
+
+#nuget {
```

Comment:
Don't think I'm seeing this id generated anywhere in the HTML. Am I missing something?

Reply:
I just pulled the base from https://github.com/fsprojects/FSharp.Formatting/blob/master/docs/files/content/style.css so anything we dont use we can just purge.